docs/lessons/kubernetes/lesson-387-a-shell-program-inside-a-manifest-is-a-second-language-nothing-validates.md

```
@@ -0,0 +1,44 @@
+owner: manu
+category: kubernetes
+tags: [kubelab, kubernetes, gitops-delivery, observability, ci-automation]
+---
+
+# A shell program embedded in a manifest is a second language, and nothing in the delivery path reads it as one
+
+**Context**: Demonstrating AC2 of #1377 on staging — the PVC alert had to be *shown* to fire when a PVC is genuinely unbound, not assumed. #1378's fix changed `disk-watcher`'s jq emitter from `healthy: (.status.phase == "Bound")` (a JSON boolean, which LogQL's `unwrap` drops) to `healthy: (if .status.phase == "Bound" then 1 else 0 end)`, and added a comment block explaining why.
+
+**Problem**: The fixed CronJob never ran. The 02:45 job failed where 02:00, 02:15 and 02:30 had all completed on the old emitter. The cause was in the comment, not the code:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)